app/pages/sections/catalogue-solutions/order-item/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,276 @@
+};
+
+const pageSetup = async (withAuth = true) => {
+  if (withAuth) {
+    mocks();
+    await setCookies();
+    await selectedRecipientIdState();
+    await selectedSolutionIdState();
+  }
+};
+
+const getLocation = ClientFunction(() => document.location.href);
+
+fixture('Catalogue-solutions - recipient page - general')
+  .page('http://localhost:1234/order/some-fake-page')
+  .afterEach(async (t) => {
+    const isDone = nock.isDone();
+    if (!isDone) {
+      nock.cleanAll();
+    }
+
+    await t.expect(isDone).ok('Not all nock interceptors were used!');
+  });
+
+test('when user is not authenticated - should navigate to the identity server login page', async (t) => {
+  nock('http://identity-server')
+    .get('/login')
+    .reply(200);
+
+  await pageSetup(false);
+  await t.navigateTo(pageUrl);
+
+  await t
+    .expect(getLocation()).eql('http://identity-server/login');
+});
+
+test('should render Catalogue-solutions order-item page', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+  const page = Selector('[data-test-id="order-item-page"]');
+
+  await t
+    .expect(page.exists).ok();
+});
+
+test('should navigate to /organisation/order-id/catalogue-solutions/select/solution/recipient when click on backlink', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const goBackLink = Selector('[data-test-id="go-back-link"] a');
+
+  await t
+    .expect(goBackLink.exists).ok()
+    .click(goBackLink)
+    .expect(getLocation()).eql('http://localhost:1234/order/organisation/order-id/catalogue-solutions/select/solution/recipient');
+});
+
+test('should render the title', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const title = Selector('h1[data-test-id="order-item-page-title"]');
+
+  await t
+    .expect(title.exists).ok()
+    .expect(await extractInnerText(title)).eql('Solution One information for Recipient 1 (recipient-1)');
+});
+
+test('should render the description', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const description = Selector('h2[data-test-id="order-item-page-description"]');
+
+  await t
+    .expect(description.exists).ok()
+    .expect(await extractInnerText(description)).eql(content.description);
+});
+
+test('should render legend with mainAdvice', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const mainAdvice = Selector('legend');
+
+  await t
+    .expect(mainAdvice.exists).ok()
+    .expect(await extractInnerText(mainAdvice)).eql(content.plannedDateQuestions[0].mainAdvice);
+});
+
+test('should render additionalAdvice', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const additionalAdvice = Selector('[data-test-id="date-field-input"] span.nhsuk-hint');
+
+  await t
+    .expect(additionalAdvice.exists).ok()
+    .expect(await extractInnerText(additionalAdvice)).eql(content.plannedDateQuestions[0].additionalAdvice);
```

Comment:
bit confused as to how these are passing. We have changed the manifest so this should be 
`content.questions.plannedDate.additionalAdvice` ??

Reply:
ah they hadn't passed it was only the unit tests I ran will fix them up